tracer/build/_build/Build.Steps.cs

```diff
@@ -596,8 +596,7 @@ void CopyWaf(string architecture, IEnumerable<string> frameworks, AbsolutePath a
         {
             if (IsWin)
             {
-                CompressZip(TracerHomeDirectory, WindowsTracerHomeZip, fileMode: FileMode.Create);
-                // for now we do not need a monitoring-home.zip file. So no need to create it.
+                CompressZip(MonitoringHomeDirectory, WindowsTracerHomeZip, fileMode: FileMode.Create);
```

Comment:
I have a question for the reviewers: should we include the profiler in the `windows-tracer-home.zip` file or should we add another zip file `monitoring-home.zip` which will contain the profiler and the tracer ?